dms_field/i18n/dms_field.pot

```diff
@@ -164,7 +166,6 @@ msgstr ""
 #: model:ir.model.fields,field_description:dms_field.field_decimal_precision__dms_directory_ids
 #: model:ir.model.fields,field_description:dms_field.field_dms_access_group__dms_directory_ids
 #: model:ir.model.fields,field_description:dms_field.field_dms_category__dms_directory_ids
-#: model:ir.model.fields,field_description:dms_field.field_dms_classification_template__dms_directory_ids
```

Comment:
Please remove all these changes from the files, as you locally don't have all the modules installed. Anyway, this makes me wonder if we should have this field populated for all the models:

https://github.com/OCA/dms/blob/d926f50e5e41a848d591cd7b28382da9cef040a1/dms_field/models/base.py#L11

Reply:
Changes removed. IMO I think that `dms_directory_ids` field should exist in `dms`, but I think it's something that can be discussed (along with other `dms_field` stuff) in the 16.0 migration.

Reply:
OK, I ping @etobella for his knowledge.